Tests/View/ViewTest.php

```diff
@@ -46,7 +46,7 @@ public function testSetLocation()
 
         $view = View::createRedirect($url, $code);
         $this->assertAttributeEquals($url, 'location', $view);
-        $this->assertAttributeEquals(null, 'route', $view);
+        $this->assertAttributeEmpty('route', $view);
```

Comment:
I am not sure about this. Empty is not necessarily the same as `null`.

Reply:
Gonna rollback this changes so :+1: